features/net/FEATURE_IPV4/lwip-interface/EthernetInterface.cpp

```diff
@@ -19,9 +19,32 @@
 
 
 /* Interface implementation */
+EthernetInterface::EthernetInterface()
+    : _dhcp(false), _ip_address(""), _netmask(""), _gateway("")
+{
+}
+
+int EthernetInterface::set_network(const char *ip_address, const char *netmask, const char *gateway)
+{
+    _dhcp = false;
+    strncpy(_ip_address, ip_address ? ip_address : "", sizeof _ip_address);
```

Comment:
I know it's not necessary, but we always seem to be using parenthesis after sizeof, could we keep to that.